### PR TITLE
Add department model

### PR DESCRIPTION
**Pull request by Bohdanvakh**

### CODE REVIEW

File: db/migrate/20230509162350_create_departments.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    create_table :departments do |t|
+      t.string :name
+      t.string :abbreviation
+      t.integer :manager_id
+
+      t.timestamps
+    end
+  end
+  def down
+    drop_table :departments
```
Comment (kvasyaa) [inline db/migrate/20230509162350_create_departments.rb]: no need for `down`. `create_table` is reversible.

File: db/migrate/20230509162350_create_departments.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class CreateDepartments < ActiveRecord::Migration[7.0]
+  def up
+    create_table :departments do |t|
+      t.string :name
+      t.string :abbreviation
+      t.integer :manager_id
```
Comment (kvasyaa) [inline db/migrate/20230509162350_create_departments.rb]: it should be a reference

**Review by kvasyaa — CHANGES_REQUESTED** [review]

File: db/migrate/20230509162350_create_departments.rb (hunk 2 above)
Comment (Bohdanvakh) [reply db/migrate/20230509162350_create_departments.rb]: I'm not sure about this attribure. Should we keep managers in a separate model or we can take employee from Employee model as a manager?

Comment (Bohdanvakh) [reply db/migrate/20230509162350_create_departments.rb]: or it should be just a record in Department model without any ID and info?

**Review by kvasyaa — CHANGES_REQUESTED** [review]
Let's remove `manager_id` column for now. Because you have to mention it in the model like `belongs_to :manager, class: 'Employee'`. SInce you don't have it, it will raise an error.

**Review by kvasyaa — APPROVED** [review]

**Pull request merged**